src/app/tests/suites/certification/Test_TC_DESC_2_1.yaml

```diff
@@ -246,5 +246,20 @@ tests:
     - label: "Step 5: TH reads from the DUT the 'TagList' attribute."
       PICS: DESC.S.A0004
       verification: |
-          SDK has to implement the Taglist attribute
+          ./chip-tool descriptor read  parts-list 1 0
```

Comment:
This is misleading: it implies that the value from the sample is always the value that should be there. There isn't always a need for a TagList. This will lead to possibly testers looking for these values when any values are usable.

This test should 100% be automated.

Reply:
Fixes.

Reply:
@cecille is working on TC-DESC-2.2 which will cover this. Please add a comment that this will be replaced by TC-DESC-2.2